### PR TITLE
update sfe

### PR DESCRIPTION
**Pull request by jmyorston**
updated sfe links, fixed product sidebar for sfe

### CODE REVIEW

File: docs/sync-for-expenses/configandcategorize.md [hunk 1]
```diff
@@ -28,16 +28,16 @@ Using this endpoint you will be able to:
...
-> 1). Each Foreign Exchange currency has its own bank account
->
-> 2). Each transaction is converted back to the currency of the bank account
-> 
-> Today Sync for Exenses only supports option 2.
+:::info Foreign Exchange 💱 
 
+There are two ways to handle transactions in a foreign currency
+
+1). Each Foreign Exchange currency has its own bank account
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:35]: Please use full stops after numbers in lists. See the specification for GitHub Flavored Markdown:

https://github.github.com/gfm/#ordered-list-marker

File: docs/sync-for-expenses/configandcategorize.md [hunk 2]
```diff
@@ -28,16 +28,16 @@ Using this endpoint you will be able to:
...
-> There are two ways to handle transactions in a foreign currency
-> 
-> 1). Each Foreign Exchange currency has its own bank account
->
-> 2). Each transaction is converted back to the currency of the bank account
-> 
-> Today Sync for Exenses only supports option 2.
+:::info Foreign Exchange 💱 
 
+There are two ways to handle transactions in a foreign currency
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:33]:
```suggestion
There are two ways to handle transactions in a foreign currency:
```

File: docs/sync-for-expenses/configandcategorize.md [hunk 3]
```diff
@@ -28,16 +28,16 @@ Using this endpoint you will be able to:
...
+
+1). Each Foreign Exchange currency has its own bank account
+
+2). Each transaction is converted back to the currency of the bank account
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:37]:
```suggestion
2). Each transaction is converted back to the currency of the bank account.
```

File: docs/sync-for-expenses/configandcategorize.md [hunk 4]
```diff
@@ -28,16 +28,16 @@ Using this endpoint you will be able to:
...
+
+Today Sync for Expenses only supports option 2.
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:39]:
```suggestion
Sync for Expenses currently supports option 2 only.
```

File: docs/sync-for-expenses/sync-process/syncing-expenses.md [hunk 5]
```diff
@@ -9,9 +9,10 @@ Once you have pushed data to Codat and created datasets you can then initiate th
 The sync process includes fetching the transactional datasets from the data cache, validating and mapping the data, and then pushing it to the accounting platform. 
 
 
-<!-- theme: info -->
-> Sync's are independent of creating datasets, so you can continue to create new datasets whilst a sync is ongoing.
-
+:::tip Datasets
+Sync's are independent of creating datasets, so you can continue to create new datasets whilst a sync is ongoing.
```
Comment (d-laing) [inline docs/sync-for-expenses/sync-process/syncing-expenses.md:13]:
```suggestion
Syncs are independent of creating datasets, so you can continue to create new datasets while a sync is ongoing.
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Looks great!

File: docs/sync-for-expenses/configandcategorize.md [hunk 6]
```diff
@@ -28,16 +28,16 @@ Using this endpoint you will be able to:
...
-> 1). Each Foreign Exchange currency has its own bank account
->
-> 2). Each transaction is converted back to the currency of the bank account
-> 
-> Today Sync for Exenses only supports option 2.
+:::info Foreign Exchange 💱 
 
+There are two ways to handle transactions in a foreign currency:
+
+1). Each Foreign Exchange currency has its own bank account
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:35]:
```suggestion
1. Each Foreign Exchange currency has its own bank account
```

File: docs/sync-for-expenses/configandcategorize.md [hunk 7]
```diff
@@ -28,16 +28,16 @@ Using this endpoint you will be able to:
...
+
+2). Each transaction is converted back to the currency of the bank account.
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:37]:
```suggestion
2. Each transaction is converted back to the currency of the bank account.
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Fixed the numbering. Please ensure all numbered lists use the same format, thanks.

**Review by mcclowes — APPROVED** [review]

**Pull request merged**